websites/#/アイドルマスターシャイニーカラーズ/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+	clientId: "1103931016525127792",
+});
+
+presence.on("UpdateData", async () => {
+	const { pathname } = document.location,
+		browsingTimestamp = Math.floor(Date.now() / 1000),
```

Comment:
This should probably be outside of the `UpdateData` loop, otherwise it will always be saying that the user was playing for 0 seconds.